aws-transfer-agreement/src/test/java/software/amazon/transfer/agreement/CreateHandlerTest.java

```diff
@@ -130,38 +126,31 @@ public void handleRequest_InternalServiceErrorExceptionFailed() {
 
         assertThrows(CfnServiceInternalErrorException.class, () -> {
             handler.handleRequest(proxy, request, null, logger);
-        } );
+        });
     }
 
     @Test
     public void handleRequest_ResourceExistsExceptionFailed() {
         CreateHandler handler = new CreateHandler(client);
 
-        doThrow(ResourceExistsException.class)
-                .when(proxy)
-                .injectCredentialsAndInvokeV2(any(), any());
+        doThrow(ResourceExistsException.class).when(proxy).injectCredentialsAndInvokeV2(any(), any());
 
-        ResourceModel model = ResourceModel.builder()
-                .agreementId(TEST_AGREEMENT_ID)
-                .serverId(TEST_SERVER_ID)
-                .build();
+        ResourceModel model = ResourceModel.builder().serverId(TEST_SERVER_ID).build();
```

Comment:
This is the only code change.

Reply:
Never mind, I had to change this back to include the agreement ID. I don't know why but for some reason running `mvn verify` fails without the agreement ID, yet tests pass internally without the agreement ID here. 

```
[ERROR] software.amazon.transfer.agreement.CreateHandlerTest.handleRequest_ResourceExistsExceptionFailed -- Time elapsed: 0.008 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <software.amazon.cloudformation.exceptions.CfnAlreadyExistsException> but was: <java.lang.NullPointerException>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at software.amazon.transfer.agreement.CreateHandlerTest.handleRequest_ResourceExistsExceptionFailed(CreateHandlerTest.java:144)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.NullPointerException: Cannot invoke "org.json.JSONObject.toString()" because the return value of "software.amazon.transfer.agreement.ResourceModel.getPrimaryIdentifier()" is null
	at software.amazon.transfer.agreement.CreateHandler.handleRequest(CreateHandler.java:87)
	at software.amazon.transfer.agreement.CreateHandlerTest.lambda$handleRequest_ResourceExistsExceptionFailed$2(CreateHandlerTest.java:145)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
	... 6 more
```